website/docs/docs/dbt-versions/release-notes.md

```diff
@@ -20,6 +20,7 @@ Release notes are grouped by month for both multi-tenant and virtual private clo
 
 ## October 2024
 
+- **Enhancement**: In dbt Cloud Versionless, you can now specify snapshot configurations in YAML files, including `schema.yml` or `dbt_project.yml`, for snapshots defined in SQL files. This update resolves the previous limitation that required snapshot properties to be defined exclusively in the SQL file. This enhancement will soon be released in the upcoming dbt Core v1.9 release
```

Comment:
@dbeatty10 added a release note entry for this - can yo triple check

Reply:
```suggestion
- **Enhancement**: In dbt Cloud Versionless, snapshots defined in SQL files can now use `config` defined in `schema.yml` YAML files. This update resolves the previous limitation that required snapshot properties to be defined exclusively in `dbt_project.yml` and/or a `config()` block within the SQL file. This enhancement will be included in the upcoming dbt Core v1.9 release.
```